build-gluster-org/scripts/debian-ubuntu-package.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+#for both the debian and ubuntu packaging of various versions we have to
+#ssh to a particular ubuntu machine and package it using pbuilder.
+
+set -xe
+
+if [ $# -eq 5 ]; then
+        echo "building everything"
+        echo "packing debian distribution"
+        ../scripts/generic_package.sh debian stretch $SERIES $VERSION $RELEASE $LATEST_SERIES $LATEST_VERSION
```

Comment:
I think using ../scripts will not work. This will depend on the working directory jenkins use, and I suspect this will not be /opt/qa. better use the absolute path.

Also, I think a for loop will be easier to read.

Reply:
done